melodies_monet/util/write_util.py

```diff
@@ -4,6 +4,49 @@
 import numpy as np
 from pandas.api.types import is_float_dtype
 
+def write_grouped_ncf(obj, output_name, title=''):
```

Comment:
Seems like this is potentially writing more than one nc file but using the same name for all. Might be simpler just to default to using the key for the name (at least the stem, e.g. `airnow`, `airnow_wrfchem`). Could offer an option to apply a suffix to all of the stems to allow for some control.

Reply:
Update: @colin-harkins has informed me that this writes a single file using netCDF4 [groups](https://unidata.github.io/netcdf4-python/#groups-in-a-netcdf-file).